src/database/migrations/20190907090013-add_role_to_user.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  up: (queryInterface, Sequelize) => Promise.all([queryInterface.addColumn('Users', 'role', {
+    type: Sequelize.INTEGER,
```

Comment:
Instead of having 2 migration files to remove a column and add it, we can have one migration that alters the datatype of the column.